src/foundations/theme-provider/themes/theme.light.ts

```diff
@@ -88,6 +88,9 @@ const lightTheme: TTheme = {
         shimmer: neutral[0],
         contrast: neutral[1100],
     },
+    highlight: {
```

Comment:
this could go into the `background` property as well, right?

Reply:
I think not because there will be different highlights later on, like pinned texts, I know they change the bg color but the shades are quite different. WDYT?

Reply:
could be ... it's fine for now I think. We can always change the structure/behavior of themes later on